packages/core/src/ceramic.ts

```diff
@@ -117,12 +117,18 @@ class Ceramic implements CeramicApi {
                private _validateDocs: boolean = true,
                docCacheLimit = DEFAULT_BASE_DOC_CACHE_LIMIT,
                cacheDocumentCommits = true) {
-    this._docCache = new DocCache(docCacheLimit, cacheDocumentCommits)
     this._doctypeHandlers = {
       'tile': new TileDoctypeHandler(),
       'caip10-link': new Caip10LinkDoctypeHandler()
     }
 
+    // on evict the Document is being unregistered from the Dispatcher and removed from cache
+    // also, the event 'evicted' is fired so the client can catch the eviction and possibly load the doc again
+    this._docCache = new DocCache(docCacheLimit, async (d: Document) => {
+      d.doctype.emit('evicted');
```

Comment:
I thought we decided on the call earlier that we weren't going to add this event, and we'll just temporarily let Documents silently stop receiving updates if they get evicted from the cache, until we can do the fuller/better thing that requires the refactoring

Reply:
the PR was created before the call. It is removed now